### PR TITLE
FAT2-74 FAT2-73 course list and course details

### PR DESCRIPTION
**Pull request by dashton82**
This also includes a lot of FAT2-73 as well apart from the view

### CODE REVIEW

File: src/SFA.DAS.FAT.Application.UnitTests/SFA.DAS.FAT.Application.UnitTests.csproj [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="FluentAssertions" Version="5.10.3" />
+        <PackageReference Include="Microsoft.Extensions.Options" Version="2.2.0" />
+        <PackageReference Include="nunit" Version="3.12.0" />
+        <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
+        <PackageReference Include="SFA.DAS.Testing.AutoFixture" Version="3.0.106" />
+        <PackageReference Include="System.ComponentModel.TypeConverter" Version="4.3.0" />
```
Comment (scottywakefield) [inline src/SFA.DAS.FAT.Application.UnitTests/SFA.DAS.FAT.Application.UnitTests.csproj:16]: what's this for?

File: src/SFA.DAS.FAT.Infrastructure.UnitTests/SFA.DAS.FAT.Infrastructure.UnitTests.csproj [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="FluentAssertions" Version="5.10.3" />
+        <PackageReference Include="Microsoft.Extensions.Options" Version="2.2.0" />
+        <PackageReference Include="nunit" Version="3.12.0" />
+        <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
+        <PackageReference Include="SFA.DAS.Testing.AutoFixture" Version="3.0.106" />
+        <PackageReference Include="System.ComponentModel.TypeConverter" Version="4.3.0" />
```
Comment (scottywakefield) [inline src/SFA.DAS.FAT.Infrastructure.UnitTests/SFA.DAS.FAT.Infrastructure.UnitTests.csproj:16]: not needed

File: src/SFA.DAS.FAT.Web.AcceptanceTests/SFA.DAS.FAT.Web.AcceptanceTests.csproj [hunk 3]
```diff
@@ -22,4 +22,8 @@
     <ProjectReference Include="..\SFA.DAS.FAT.Web\SFA.DAS.FAT.Web.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Include="Features\Home.feature" />
+  </ItemGroup>
```
Comment (scottywakefield) [inline src/SFA.DAS.FAT.Web.AcceptanceTests/SFA.DAS.FAT.Web.AcceptanceTests.csproj:27]: shouldn't need this

File: src/SFA.DAS.FAT.Web.UnitTests/SFA.DAS.FAT.Web.UnitTests.csproj [hunk 4]
```diff
@@ -7,9 +7,17 @@
...
-        <PackageReference Include="nunit" Version="3.11.0"/>
-        <PackageReference Include="NUnit3TestAdapter" Version="3.11.0"/>
-        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0"/>
+        <PackageReference Include="FluentAssertions" Version="5.10.3" />
+        <PackageReference Include="Microsoft.Extensions.Options" Version="2.2.0" />
+        <PackageReference Include="nunit" Version="3.12.0" />
+        <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
+        <PackageReference Include="SFA.DAS.Testing.AutoFixture" Version="3.0.106" />
+        <PackageReference Include="System.ComponentModel.TypeConverter" Version="4.3.0" />
```
Comment (scottywakefield) [inline src/SFA.DAS.FAT.Web.UnitTests/SFA.DAS.FAT.Web.UnitTests.csproj:16]: ??

File: src/SFA.DAS.FAT.Web.AcceptanceTests/SFA.DAS.FAT.Web.AcceptanceTests.csproj (hunk 3 above)
Comment (dashton82) [reply src/SFA.DAS.FAT.Web.AcceptanceTests/SFA.DAS.FAT.Web.AcceptanceTests.csproj:27]: sadly needed for rider 😢

File: src/SFA.DAS.FAT.Infrastructure.UnitTests/SFA.DAS.FAT.Infrastructure.UnitTests.csproj (hunk 2 above)
Comment (dashton82) [reply src/SFA.DAS.FAT.Infrastructure.UnitTests/SFA.DAS.FAT.Infrastructure.UnitTests.csproj:16]: To fix build warnings!

**Pull request merged**